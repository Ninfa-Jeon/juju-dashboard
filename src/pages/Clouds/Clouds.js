import React from "react";

import Layout from "components/Layout/Layout";

export default function Clouds() {
  return (
    <Layout>
      <div className="p-strip">
        <div className="row">
          <h2>Clouds</h2>
        </div>
      </div>
    </Layout>
  );
}
