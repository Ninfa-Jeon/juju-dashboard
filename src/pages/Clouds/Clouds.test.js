import React from "react";
import { shallow } from "enzyme";
import Clouds from "./Clouds";

describe("Clouds page", () => {
  it("renders without crashing and matches snapshot", () => {
    const wrapper = shallow(<Clouds />);
    expect(wrapper).toMatchSnapshot();
  });
});
