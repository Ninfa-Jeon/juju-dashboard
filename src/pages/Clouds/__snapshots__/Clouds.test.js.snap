// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clouds page renders without crashing and matches snapshot 1`] = `
<Layout
  sidebar={true}
>
  <div
    className="p-strip"
  >
    <div
      className="row"
    >
      <h2>
        Clouds
      </h2>
    </div>
  </div>
</Layout>
`;
